atomic_reactor/util.py

```diff
@@ -742,20 +748,29 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     :param image: ImageName, the remote image to inspect
     :param registry: str, URI for registry, if URI schema is not provided,
                           https:// will be used
+    :param digest: str, digest of the image manifest
     :param insecure: bool, when True registry's cert is not verified
     :param dockercfg_path: str, dirname of .dockercfg location
-    :param versions: tuple, which manifest schema versions to fetch digest
+    :param version: str, which manifest schema versions to fetch digest
 
     :return: dict, versions mapped to their digest
     """
     response = query_registry(
         image, registry, digest=digest, insecure=insecure,
         dockercfg_path=dockercfg_path, version=version)
     response.raise_for_status()
+    logger.debug("response json: {}".format(response.json()))
+    config_digest = response.json()['config']['digest']
+
+    config_response = query_registry(
+        image, registry, digest=config_digest, insecure=insecure,
+        dockercfg_path=dockercfg_path, version=version, is_blob=True)
+    config_response.raise_for_status()
+    logger.debug("response json: {}".format(config_response.json()))
```

Comment:
landscape suggests using `logging.debug("response json: %s". config_response.json())`. Could even do `config = response.json()` since we are using the result immediately.